### ISSUE TITLE
Error when using happypack

### ISSUE THREAD

**Issue by Whoaa512** [stephencookdev/speed-measure-webpack-plugin#20]
Just tried adding this and running and got this error: 
```
ERROR in ./src/apps/3dviewer/webpack-entry.js
Module build failed: AssertionError: HappyPack: plugin for the loader '1' could not be found! Did you forget to add it to the plugin list?
    at Object.HappyLoader (/Users/cjw/work/fictiv-js/node_modules/happypack/lib/HappyLoader.js:37:3)
 @ multi ./src/lib/baseConfig ./src/apps/3dviewer/webpack-entry.js
```
I can try to add a more complete set of config sometime this weekend or monday.

**Comment by stephencookdev** [comment]
Ahh, that's annoying. I'll take a look at this tomorrow - thanks for raising :slightly_smiling_face:

**Comment by stephencookdev** [comment]
Okay, this should be fixed now with 2cf8023e1553edee4d441fcbf9f30a195ec26e18 (v1.0.2)

Can you confirm whether or not this is working for you?

**Comment by Whoaa512** [comment]
Yup that works now thanks
